activestorage/app/models/active_storage/variation.rb

```diff
@@ -58,6 +58,10 @@ def key
     self.class.encode(transformations)
   end
 
+  def digest
+    Digest::SHA1.base64digest Marshal.dump(transformations)
```

Comment:
Is this consistent across Ruby versions now? Recall some issues with older Ruby where Marshal changes broke e.g. cache keys.

Reply:
It’s not less consistent than the key we were using before, which indirectly went through `Marshal.dump` (by way of `ActiveSupport::MessageVerifier`).

Reply:
👍🏼